packages/frontend-core/src/components/ClientAppSkeleton.svelte

```diff
@@ -0,0 +1,346 @@
+<script>
+  export let sideNav = false
+  export let hideDevTools = false
+  export let hideFooter = false
+  export let noAnimation = false
+</script>
+
+<svelte:head>
```

Comment:
Were using [container queries](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_containment/Container_queries) here to simulate screen resizing for loaders in the builder when there's no iframe. Unfortunately the svelte preprocessor doesn't support them in the svelte version were on, so they need to be inserted in this way. I put the media query version alongside them as it looked neater also imo.